isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/nutzungsvorgaben/inhalt.adoc

```diff
@@ -56,34 +162,58 @@ public class OAuth2WebClientConfiguration {
     @Bean("webclient")
     WebClient webClient(ClientRegistrationRepository clientRegistrationRepository,
         OAuth2AuthorizedClientService authorizedClientService) {
-        ServletOAuth2AuthorizedClientExchangeFilterFunction oauth2 =
+        ServletOAuth2AuthorizedClientExchangeFilterFunction oauth =
             new ServletOAuth2AuthorizedClientExchangeFilterFunction(
                 new AuthorizedClientServiceOAuth2AuthorizedClientManager(clientRegistrationRepository,
                     authorizedClientService));
         oauth.setDefaultClientRegistrationId("registrationId");
-        return WebClient.builder().apply(oauth2.oauth2Configuration()).build();
+        return WebClient.builder().filter(oauth).build();
     }
 
 }
 ----
 
-[[konfigurationsparameter]]
-=== Konfigurationsparameter
+[[manual-authentication]]
+=== Manuelle Authentifizierung eines Clients mit dem Authentifizierungsmanager
```

Comment:
With ISY-147 the _manual authentification_ classes and names are removed, because there will be no manual or user interactiv authentication process.
there exists only a difference between:
- **internal authentification** of confidential client applications
- **external authentification** of external applications